### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by n0v3xx** [meetfranz/franz#2126]
361 open issues and no responses from DEVs. Is that project dead?

YES ? / NO ?

**Comment by zaksquatch** [comment]
It may be time to look for an alternative, I believe one or two projects spun off of Franz?

**Comment by Gaboris** [comment]
> It may be time to look for an alternative, I believe one or two projects spun off of Franz?

Yes. Ferdi is a direct fork of Franz and Hamsket is based off of Rambox with the same base principle.

https://github.com/getferdi/ferdi

https://github.com/TheGoddessInari/hamsket

**Comment by ManuelReschke** [comment]
@Gaboris , thanks. Ferdi looks great. I will give it a try. (https://getferdi.com/)

**Comment by hutber** [comment]
How can it be dead when we pay for the product :O

**Comment by Gaboris** [comment]
Well I can't really answer you that. The subscription was I guess manly for the higher server functionality, but it has shown to be a detriment to the system whenever any outage outright cut the users' access to the whole client.

This is one of the reasons why I moved to Ferdi, but really the straw that broke the camel's back for me was the inactivity and then the removal of spell-check from the "free version".

**Comment by hutber** [comment]
Ye, that is fair. I am certainly going to check that out. But only because of this amazing bug which sends messages to other services without my knowledge. Aka... I'm typing in whatsapp, but the message is sent to facebook... However I do have spellcheck on the free version 
![image](https://user-images.githubusercontent.com/1344898/152450626-a35b087b-a107-4dbe-a8bd-ef67008c2d5f.png)

**Comment by Gaboris** [comment]
Yes I think spellcheck was added back at some point, but I have lost faith by then I'm afraid. 😩

**Comment by n0v3xx** [comment]
Over one week no response from a DEV to this topic said all.

THIS PROJECT IS DEAD.

Thanks for the alternatives, i will try it.

**Comment by DalderupMaurice** [comment]
> Over one week no response from a DEV to this topic said all.
> 
> THIS PROJECT IS DEAD.
> 
> Thanks for the alternatives, i will try it.

I actually came back to Franz because Ferdi kept logging me out of my discord on every restart. 
For me, that is unacceptable behavior.

**Comment by Gaboris** [comment]
> I actually came back to Franz because Ferdi kept logging me out of my discord on every restart. For me, that is unacceptable behavior.

That's a VERY odd behavior. I wish we had some functioning error logging in these projects cuz such a problem only came up for me when I updated the client over a year ago cuz that wiped the cache back then.
But there hasn't been any problems since so it's quite sad to hear something so random can just hit a user this consistently.

**Comment by n0v3xx** [comment]
@DalderupMaurice , i use discord on a daily basis. I had no login issues so far with Ferdi. I use Ferdi on ubuntu 20.

**Comment by DalderupMaurice** [comment]
> @DalderupMaurice , i use discord on a daily basis. I had no login issues so far with Ferdi. I use Ferdi on ubuntu 20.



> > I actually came back to Franz because Ferdi kept logging me out of my discord on every restart. For me, that is unacceptable behavior.
> 
> That's a VERY odd behavior. I wish we had some functioning error logging in these projects cuz such a problem only came up for me when I updated the client over a year ago cuz that wiped the cache back then. But there hasn't been any problems since so it's quite sad to hear something so random can just hit a user this consistently.

Yeah, I saw that people got this issue since auto-updating to 5.7.0 and many people fixed it by manually re-installing it.

But I've never had an earlier version than 5.7.0 and I just downloaded it from GH. Tried several solutions but with every restart, all my discord accounts are logged out and I have to sign in (with 2FA) on all my accounts. Really sucks as it would be the perfect solution otherwise

**Comment by Gaboris** [comment]
> > That's a VERY odd behavior. I wish we had some functioning error logging in these projects cuz such a problem only came up for me when I updated the client over a year ago cuz that wiped the cache back then. But there hasn't been any problems since so it's quite sad to hear something so random can just hit a user this consistently.
> 
> Yeah, I saw that people got this issue since auto-updating to 5.7.0 and many people fixed it by manually re-installing it.
> 
> But I've never had an earlier version than 5.7.0 and I just downloaded it from GH. Tried several solutions but with every restart, all my discord accounts are logged out and I have to sign in (with 2FA) on all my accounts. Really sucks as it would be the perfect solution otherwise

Hmmm... So you use multiple Discord accounts AND have 2FA on all of them? Maybe that's the fault?
That would be a super debilitating bug. Real shame. 😩

**Comment by DalderupMaurice** [comment]
> > > That's a VERY odd behavior. I wish we had some functioning error logging in these projects cuz such a problem only came up for me when I updated the client over a year ago cuz that wiped the cache back then. But there hasn't been any problems since so it's quite sad to hear something so random can just hit a user this consistently.
> > 
> > 
> > Yeah, I saw that people got this issue since auto-updating to 5.7.0 and many people fixed it by manually re-installing it.
> > But I've never had an earlier version than 5.7.0 and I just downloaded it from GH. Tried several solutions but with every restart, all my discord accounts are logged out and I have to sign in (with 2FA) on all my accounts. Really sucks as it would be the perfect solution otherwise
> 
> Hmmm... So you use multiple Discord accounts AND have 2FA on all of them? Maybe that's the fault? That would be a super debilitating bug. Real shame. 😩

Yeah, I have no clue, I tried deleting all discords besides 1, tried older versions, manual upgrades, automatic updates, it's just not working :P

**Comment by Gaboris** [comment]
> Yeah, I have no clue, I tried deleting all discords besides 1, tried older versions, manual upgrades, automatic updates, it's just not working :P

Oh Wait! I just realized that 5.7 is the newest version (even though it was released a month ago) and I'm still on 5.6.8 because I'm using Chocolatey for updates and no one has updated that repository since then. 🤔

That's around when there was a tear in the community and a few notable members left. Also at that point they started churning out versions without any changelogs so I'll boot up my old laptop and install the new Ferdi on that. See if it happens on its own.

Also when you installed older versions did you clean all leftover Ferdi files from the system? My installation is actually riddled with old stuff they removed, but didn't automatically delete during updates.
Also-also what system are you on?

**Comment by Gaboris** [comment]
Yeah, the new version just dropped after I made my comment so that's actually promising.

But we can all agree that the last update coming from half a year ago and there being little to no communication for who knows how long WOULD raise a lot of concerns. 😅

> Yeah, I have no clue, I tried deleting all discords besides 1, tried older versions, manual upgrades, automatic updates, it's just not working :P

BTW I just checked Ferdi's page to see if anyone was having similar issues and to my horror the Issues are disabled which is a HUGE red flag in my book.
I really hope the new devs won't kill the whole project by mismanagement cuz it seems they are ready to kill the community for it for sure. 😩

**Comment by DalderupMaurice** [comment]
> Yeah, the new version just dropped after I made my comment so that's actually promising.
> 
> But we can all agree that the last update coming from half a year ago and there being little to no communication for who knows how long WOULD raise a lot of concerns. 😅
> 
> > Yeah, I have no clue, I tried deleting all discords besides 1, tried older versions, manual upgrades, automatic updates, it's just not working :P
> 
> BTW I just checked Ferdi's page to see if anyone was having similar issues and to my horror the Issues are disabled which is a HUGE red flag in my book. I really hope the new devs won't kill the whole project by mismanagement cuz it seems they are ready to kill the community for it for sure. 😩

https://github.com/meetfranz/franz/issues/1994#issuecomment-970456257
This seemed to solve my issue :P 

There's also an alternative one that I did not try:
https://github.com/meetfranz/franz/issues/1821#issuecomment-1043548826

For Ferdi, I have no clue. I also used revo uninstaller as it indeed keeps a lot of leftover files when uninstalling and I'm using W11

**Comment by milesflavel** [comment]
> Yeah, the new version just dropped after I made my comment so that's actually promising.
> 
> But we can all agree that the last update coming from half a year ago and there being little to no communication for who knows how long WOULD raise a lot of concerns. 😅
> 
> > Yeah, I have no clue, I tried deleting all discords besides 1, tried older versions, manual upgrades, automatic updates, it's just not working :P
> 
> BTW I just checked Ferdi's page to see if anyone was having similar issues and to my horror the Issues are disabled which is a HUGE red flag in my book. I really hope the new devs won't kill the whole project by mismanagement cuz it seems they are ready to kill the community for it for sure. 😩

Yeah, this is a recent situation. In my opinion, Ferdi is not worth it anymore, the dev has gone off the deep end.

They removed the issue tracker because "some debug information shared in our GitHub issue tracker may leak private data", but then yesterday they announced that Issues will be available to sponsors only for $9/month.

The person running their Twitter account is just trolling everyone and posting some "questionable" content.

Anyone know of any non-electron-based alternatives? (something on tauri perhaps?)

**Comment by tpluscode** [comment]
Oh wow, does anyone have screenshots of the twitter exchange about issue tracker. The `@getferdi` account was removed

**Comment by milesflavel** [comment]
> Oh wow, does anyone have screenshots of the twitter exchange about issue tracker. The `@getferdi` account was removed

No, but the announcement is still listed on their FAQ here https://getferdi.com/faq/why-the-issue-tracker-was-disabled/

Unfortunately that doesn't give any idea of the level of mismanagement suggested by their rogue twitter presence.

**Comment by sandrooco** [comment]
Do you mind moving ferdi related discussions to wherever (not their issues since it's gone lol) - but the franz repo certainly isn't the place.

**Comment by Gaboris** [comment]
> Do you mind moving ferdi related discussions to wherever (not their issues since it's gone lol) - but the franz repo certainly isn't the place.

Seeing how Franz has gotten a new major version update and Ferdi has actually opened up their Issues page probably due to well deserved backlash I'd say this topic could be fully closed down as to not discourage anyone from giving the project a second try.
